CONTRIBUTING.md

```diff
@@ -1,8 +1,75 @@
-## Guide
+# Contributing to GNO
 
-### Getting started with ViM
+Thank you for looking to contribute to the GNO project.
+We appreciate every open-source contribution, as it helps us improve and enhance gno for the benefit of the community.
 
-Add to your .vimrc file:
+This document outlines some basic pointers on making your future contribution a great experience. It outlines basic PR
+etiquette employed by the core gno team. It lays out coding styles, simple how-to guides and tools to get you up and
+running and pushing code.
+
+If you are unsure about something, please don’t hesitate to reach out for help by opening an issue or discussion.
+Likewise, if you have an idea on how to improve this guide, go for it as well.
+
+## Table of Contents
+
+- [Important Resources](#important-resources)
+- [Getting Started](#getting-started)
+    - [Environment](#environment)
+    - [Local Setup](#local-setup)
+    - [Testing](#testing)
+    - [Repository Structure](#repository-structure)
+- [How do I?](#how-do-i)
+    - [How do I submit changes?](#how-do-i-submit-changes)
+    - [How do I report a bug?](#how-do-i-report-a-bug)
+    - [How do I request a feature?](#how-do-i-request-a-feature)
+- [Style Guides](#style-guides)
+    - [Git Commit Messages](#git-commit-messages)
+    - [Go Style Guide](#go-style-guide)
+    - [Documentation Style Guide](#documentation-style-guide)
+- [Additional Notes](#additional-notes)
+    - [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+    - [Notable Contributions](#notable-contributions)
+
+## Important Resources
+
+- **[Discord](https://discord.gg/YFtMjWwUN7)** - we are very active on Discord. Join today and start discussing all
+  things gno with fellow engineers and enthusiasts.
+- **[Active Staging](https://gno.land/)** - use the currently available staging environment to play around with a
+  production network. If you want to interact with a local instance, refer to the [Local Setup](#local-setup) guide.
+- **[Twitter](https://twitter.com/_gnoland)** - follow us on Twitter to get the latest scoop
+- **[Telegram](https://t.me/gnoland)** - join our official Telegram group to start a conversation about gno
+- **[Awesome GNO](https://github.com/gnolang/awesome-gno)** - check out the list of compiled resources for helping you
+  understand the gno ecosystem
```

Comment:
```suggestion
- **[Discord](https://discord.gg/YFtMjWwUN7)** - we are very active on Discord. Join today and start discussing all things gno with fellow engineers and enthusiasts.
- **[Awesome GNO](https://github.com/gnolang/awesome-gno)** - check out the list of compiled resources for helping you understand the gno ecosystem
- **[Active Staging](https://gno.land/)** - use the currently available staging environment to play around with a
  production network. If you want to interact with a local instance, refer to the [Local Setup](#local-setup) guide.
- **[Twitter](https://twitter.com/_gnoland)** - follow us on Twitter to get the latest scoop
- **[Telegram](https://t.me/gnoland)** - join our official Telegram group to start a conversation about gno
```

Reply:
Updates the order:

[2056cb0](https://github.com/gnolang/gno/pull/469/commits/2056cb019a663b9ff61249b08e9d6529b8b7bc15)